linalg/operator.cpp

```diff
@@ -69,6 +69,16 @@ void Operator::RecoverFEMSolution(const Vector &X, const Vector &b, Vector &x)
    }
 }
 
+void Operator::FormParallelOperator(Operator* &Aout)
```

Comment:
Since this could be used for square operators, we probably should also add boundary conditions (or have two separate methods)